actionpack/test/controller/parameters/parameters_permit_test.rb

```diff
@@ -62,7 +62,7 @@ def walk_permitted(params)
     values += [0, 1.0, 2**128, BigDecimal(1)]
     values += [true, false]
     values += [Date.today, Time.now, DateTime.now]
-    values += [STDOUT, StringIO.new, ActionDispatch::Http::UploadedFile.new(tempfile: __FILE__),
+    values += [$stdout, StringIO.new, ActionDispatch::Http::UploadedFile.new(tempfile: __FILE__),
```

Comment:
I'm not sure about this one.  The intent was actually `STDOUT`, but I don't know if it makes a difference.